superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -96,7 +96,7 @@ export default class AddSliceContainer extends React.PureComponent<
           <Panel.Body>
             <div>
               <p>{t('Choose a datasource')}</p>
-              <p>
+              <div style={{ marginBottom: '10px' }}>
```

Comment:
Is this wrapper div even necessary? Or should the div within, `<div style={styleSelectWidth}>` just have this margin added there?

Reply:
You're right, it wasn't necessary. I renamed `styleSelectWidth` to `styleSelectContainer` and removed this style from `<Select>` (it wasn't necessary)